dnf-behave-tests/features/install-path.feature

```diff
@@ -15,3 +15,30 @@ Examples:
         | relative with wildcards | noarch/setup-*.fc29.noarch.rpm                                                      |
         | file://                 | file://{context.dnf.fixturesdir}/repos/dnf-ci-fedora/noarch/setup-2.12.1-1.fc29.noarch.rpm |
 
+
+Scenario: I can install an RPM from path, when specifying the RPM multiple times
+   When I execute dnf with args "install noarch/setup-2.12.1-1.fc29.noarch.rpm noarch/setup-2.12.1-1.fc29.noarch.rpm" from repo "dnf-ci-fedora"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                                   |
+        | install       | setup-0:2.12.1-1.fc29.noarch              |
+
+
+@xfail
```

Comment:
So these tests fail for install but pass for update?

Reply:
Exactly.

Reply:
Okay :slightly_smiling_face: